### PR TITLE
Add sentiment field validation for feedback model

### PR DESCRIPTION
**Pull request by onmete**
## Description

Something I noticed during Pavel's demo. We don't have an explicit check for the value. We allow either good(1) or bad(-1).

## Type of change

- [x] New feature

### CODE REVIEW

File: ols/app/models/models.py [hunk 1]
```diff
@@ -283,6 +283,14 @@ def check_uuid(cls, value: str) -> str:
             raise ValueError(f"Improper conversation ID {value}")
         return value
 
+    @field_validator("sentiment")
+    @classmethod
+    def check_sentiment(cls, value: int) -> int:
```
Comment (tisnik) [inline ols/app/models/models.py:288]: is `None` allowed?

Comment (tisnik) [reply ols/app/models/models.py:288]: so it's `Optional[int]` -> `Optional[int]`?

Comment (tisnik) [reply ols/app/models/models.py:288]: I mean, just specify that the member in model is optional, the parameter is optional, and indeed, return value is optional too ;)

Comment (onmete) [reply ols/app/models/models.py]: Ok, you are talking about typing. Updated.

**Pull request merged**